amethyst_gltf/src/format/mod.rs

```diff
@@ -0,0 +1,545 @@
+//! GLTF format
+
+use std::collections::HashMap;
+use std::error::Error as StdError;
+use std::fmt;
+use std::sync::Arc;
+
+use self::importer::{get_image_data, import, Buffers, ImageFormat};
+use assets::{BoxedErr, Format, Source};
+use core::transform::LocalTransform;
+use gfx::Primitive;
+use gfx::texture::SamplerInfo;
+use gltf;
+use gltf::Gltf;
+use itertools::Itertools;
+use renderer::vertex::*;
+
+use super::*;
+
+mod importer;
+
+/// Gltf scene format, will cause the whole default scene to be loaded from the given file.
+///
+/// Using the `GltfSceneLoaderSystem` a `Handle<GltfSceneAsset>` from this format can be attached
+/// to an entity in ECS, and the system will then load the full scene using the given entity
```

Comment:
Good idea :+1:

Reply:
This could probably be replaced by a real prefab system at some point

Reply:
More like generalized into prefab system